app/templates/database/-replicate.hbs

```diff
@@ -0,0 +1,14 @@
+<div class="panel-heading">
+  <h3>Submit a support request</h3>
+</div>
+<div class="panel-body">
+  <p>
+  Replication and clustering are currently in beta.
+  To add a new replica or cluster member,
+  <a href="{{supportPortal}}">please contact support</a>.
```

Comment:
We have a helper that encapsulates this, you can see it in action here: https://github.com/aptible/dashboard.aptible.com/blob/e2df50c16c71733f97eb7ef57a0147ac8ab46627/app/stack/activate/template.hbs#L8